elasticsearch/index.go

```diff
@@ -372,19 +372,8 @@ func (esi *Index) FilterByMatchQuery(typ string, name string, value interface{})
 
 // SearchByJSON performs a search over the index via raw JSON.
 func (esi *Index) SearchByJSON(typ string, jsn string) (*SearchResult, error) {
```

Comment:
Should i pass in a bool to allow skipping of these checks or is there a neat golang way of overriding this functionality for EventDB only?